src/tools/object_selector.cpp

```diff
@@ -82,7 +88,7 @@ bool ObjectSelector::selectAt(const MapCoordF& position, qreal tolerance, bool t
 		{
 			// Results different - select object with highest priority, if it is not the same as before
 			last_results = objects;
-			std::sort(objects.begin(), objects.end(), sortObjects);
+			sortObjects(objects);
```

Comment:
I would like to make it explicit: I want `std::sort` in these lines. 
But I acknowledge that the original `sortObject` identifier causes confusion. Maybe it should be `compareTypeAndExtent`.

Reply:
Ok, keeping `std::sort` was my preference as well but I misunderstood your "Embrace std::sort and the custom operator".